README.md

```diff
@@ -53,9 +53,7 @@ Download the [android plugin](https://github.com/kyoz/godot-haptics/releases) (m
 
 Enable `Haptics` plugin in your android export preset
 
-Enable `Haptics` permission.
```

Comment:
This permission does not exist. However, there is a `VIBRATE` permission that might be necessary for haptics to funtion.